deployment/ansible/group_vars/all

```diff
@@ -20,3 +20,4 @@ postgis_package_version: "2.1.*.pgdg14.04+1"
 nodejs_npm_version: 2.1.17
 
 temp_dir: "/media/ephemeral0"
+worker_instances: 1
```

Comment:
Might be worth changing the name of this to something more specific to EMR. I initially thought it was for number of worker process instances on a worker machine.